services/harmony/app/frontends/service-image-tags.ts

```diff
@@ -335,9 +332,7 @@ export async function updateServiceImageTag(
   const { service } = req.params;
   const { tag } = req.body;
 
-  const status = await module.exports.execDeployScript(req, res, service, tag);
-  if (status == successfulStatus) {
-    res.statusCode = 201;
-    res.send({ 'tag': tag });
-  }
+  await module.exports.execDeployScript(req, res, service, tag);
```

Comment:
As I mentioned above I think you can remove the `async` from the `execDeployScript` definition and then remove the `await` here.